javascript/javascript2/week3/fastestPresser.js

```diff
@@ -0,0 +1,56 @@
+const timeinput = document.getElementById("startInput");
+const startBtn = document.getElementById("start");
+const player1 = document.getElementById("player1");
+const player2 = document.getElementById("player2");
+// prevent the user to type in input field
+timeinput.onkeydown = (e) => {
+  e.preventDefault;
+  return false;
+};
+
+let totlaNumberOfSeconds;
+
+let player1PressCount = 0;
+let player2PressCount = 0;
+
+timeinput.addEventListener("input", () => {
+  totlaNumberOfSeconds = timeinput.value;
+});
```

Comment:
You have access to the `event` on the callback here: 
```suggestion
timeinput.addEventListener("input", (event) => {
  totlaNumberOfSeconds = timeinput.value;
});
```

Try and grab the value of the input from there. It will make your code more maintainable and easier to read!